### ISSUE TITLE
[Bug]: Whitelisting doesn't match example in whitelist-sample.txt

### ISSUE THREAD

**Issue by ghost** [toniblyx/prowler#1059]
**What happened?**  
Regex whitelisting no longer works, only basic bash globbing. 

```bash
-    if [[ $1 =~ ${resource_value} ]]; then
+    # To set WARNING flag both values must be exactly the same
+    if [[ "${checked_value}" == *"${resource_value}"* ]]; then
```
3097ba6c667ba0e21a7923efac1c49c88abb117d
30ce25300f1f07a54ac889b1bfe4856adb84f42c

"==" limits bash to globbing.
Quoting the variable means to interpret the variable literally (ie "exactly the same")

**How to reproduce it**  
1. What command are you running?
Create a bucket `aws s3 mb s3://foobarbaz12345-logs`
Run prowler with the sample whitelist and uncomment extra718 check
`./prowler -w whitelist-sample.txt -r <region> -f <region> -c extra718`
2. Environment you have, like single account, multi-account, organizations, etc.
Single account

3. See error
`FAIL! <region>: Bucket foobarbaz12345-logs has server access logging disabled!`

**Expected behavior**  
This should be a warning as it matches the sample whitelist  "extra718:[[:alnum:]]+-logs"

**From where are you running Prowler?**  
Please, complete the following information:
 - Resource: non-aws Linux server
 - OS: CentOS
 - AWS-CLI Version [`aws --version`]: `aws-cli/2.4.22 `
 - Prowler Version `Prowler 2.7.0-24January2022`
 - Shell and version:
 - Others:

**Additional context**

Please either revert the change or update the documentation to remove references to regex.

**Comment by n4ch04** [comment]
Hi @wobblesprout we have reformulated the logic to include POSIX character classes, please check it out here prowler-cloud/prowler#1061 and let us know if everything is ok

**Comment by ghost** [comment]
Thanks for following this up so quickly. I've added a comment to the PR.

**Comment by n4ch04** [comment]
Thanks for your suggestion ! It's the same coded in a different way
We have scheduled the reformulation of whitelisting logic, including this, we just have changed `ignore_check_name` to follow the same syntax as `resource_value`, please check it again and let us know if works like it should

**Comment by n4ch04** [comment]
We have been merged prowler-cloud/prowler#1061, please @wobblesprout feel free to reopen if a new error appears, and thank you so much for your help and advise !!
